item-spec/json-schema/item.json

```diff
@@ -203,6 +203,18 @@
             "title": {
               "title": "Link title",
               "type": "string"
+            },
+            "method": {
+              "title": "Link method",
+              "type": "string"
```

Comment:
Should this be an enum?

Reply:
@emmanuelmathot I think we should provide an enum here so that it's (1) clear which ones are allowed and (2) whether they are uppercase or lowercase (I think it's all uppercase).